### ISSUE TITLE
Ability to change hyperlink

### ISSUE THREAD

**Issue by ekaterinashenets** [nhn/tui.editor#2359]
## Version
^3.1.2

## Current Behavior
After adding a hyperlink there's no ability to edit it.

## Expected Behavior
User should be able to edit the hyperlink that was added

**Comment by Alyw234237** [comment]
I agree that this feature would be very nice to have! Issue #2159 is relevant here and a possible duplicate. @js87zz added that issue to the "Planned Feature(3.x)" milestone a month ago so I think he might be working on it (or on functionality to facilitate it).
